api/models/recipe.model.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+const mongoosePaginate = require('mongoose-paginate');
+
+let RecipeSchema = new mongoose.Schema({
+    name: String,
```

Comment:
None of these are marked as `required`. Mark the required ones as such so we don't create empty recipes